src/usaxs/usaxs_flyscan_support/nexus_flyscan.py

```diff
@@ -210,61 +214,70 @@ def unconnected_signals(self):
             list: List of PV_Specification objects for unconnected signals
         """
         disconnects = [
-            pv for pv in self.pv_registry.values() if not pv.ophyd_signal.connected
+            pv
+            # .
+            for pv in self.pv_registry.values()
+            if not pv.ophyd_signal.connected
         ]
         return disconnects
 
-    def get_hdf5_path(self, xml_node):
-        """Get the HDF5 path for a given XML node.
+    # TODO: Will these be used?
```

Comment:
Be sure to inspect if these lines should not be commented out?  They were not in the pre-BITS code base.